doc/build.sh

```diff
@@ -48,6 +48,8 @@ EOF
 
 # Rebuild all docs versions
 for d in docs/*; do
+    d=$(basename "${d}")
+
     # don't rebuild archive dir
```

Comment:
lines 53 - 60 don't work now 
<img width="185" alt="Screen Shot 2020-03-18 at 7 54 47 AM" src="https://user-images.githubusercontent.com/5317992/76974084-009cc280-68ee-11ea-8cb5-6aba553acc04.png">

It should exclude `docs/` and the `master` and `archive` dirs:

<img width="153" alt="Screen Shot 2020-03-18 at 7 57 39 AM" src="https://user-images.githubusercontent.com/5317992/76974292-3f327d00-68ee-11ea-8723-77ee9a84339a.png">